pkg/gce-pd-csi-driver/controller.go

```diff
@@ -143,6 +166,16 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		if err != nil {
 			return nil, status.Error(codes.AlreadyExists, fmt.Sprintf("CreateVolume disk already exists with same name and is incompatible: %v", err))
 		}
+
+		ready, err := isDiskReady(existingDisk)
```

Comment:
Can we add some unit tests for the various scenarios? The fake can probably be extended so that we can toggle the ready status.

Reply:
Done